sites/partners/src/pages/listings/[id]/applications/index.tsx

```diff
@@ -182,7 +182,7 @@ const ApplicationsList = () => {
                   <div className="flex gap-2 items-center">
                     <Button
                       onClick={() => {
-                        if (listingDto.lotteryLastRunAt) {
+                        if (process.env.showLottery === "TRUE" && !!listingDto.lotteryStatus) {
```

Comment:
We do have an `errored` status for the lottery that we might want to ignore for this.
Also, for partners/jadmins that don't know the lottery has been run yet, would this message be confusing?

Reply:
I think partners/jadmins would still need to know, but also in Doorway I don't think partners/jadmins will be adding applications

Agreed on ignoring the `errored` state! When does the lottery end up in `errored` as opposed to `null`?

Reply:
In the rare case that when the lottery run is triggered something goes wrong and fails to complete it would be set to `errored` and would need to be rerun. At least that was my understanding @YazeedLoonat